src/providers/currencyConverter.ts

```diff
@@ -77,15 +82,15 @@ const currencyConverter = async (
     const toCurrency = pair.currency_pair.split('_')[1]
     if (
       isFiatCode(fromCurrency) &&
-      fromCurrency !== 'iso:USD' &&
+      fromCurrency !== DEFAULT_FIAT &&
       datesAndCodesWanted[pair.date].indexOf(`${subIso(fromCurrency)}_USD`) ===
```

Comment:
This can be something like `${subIso(fromCurrency)}_${subIso(DEFAULT_FIAT)}` no?